pmemcheck/pmemcheck.h

```diff
@@ -79,7 +79,8 @@ typedef
        VG_USERREQ__PMC_REMOVE_FROM_TX_N,
        VG_USERREQ__PMC_ADD_THREAD_TO_TX_N,
        VG_USERREQ__PMC_REMOVE_THREAD_FROM_TX_N,
-       VG_USERREQ__PMC_ADD_TO_GLOBAL_TX_IGNORE
+       VG_USERREQ__PMC_ADD_TO_GLOBAL_TX_IGNORE,
+       VG_USERREQ__PMC_DEFAULT_REORDER
```

Comment:
comma on the last line - you won't need to modify this line when someone will add new request